src/components/ContentSidebar/Sidebar.js

```diff
@@ -118,12 +121,12 @@ const Sidebar = ({
         />
     );
 
-    if (!hasActivityFeed) {
+    if (!hasActivityFeed || !currentUser) {
```

Comment:
Not sure about this, I think this check is about props that the parent app has passed in, not the success/failure of API calls.

Reply:
+1 to Jeremy's comment. We should still render activity feed, but it should be loading

Reply:
Done!